### DISCUSSION TITLE
발표 방법 정리

### DISCUSSION THREAD

**Discussion by Sleepingoff** [OurStudyLog/operatingsystem#5] [Ideas]
시간 제한을 두고 발표 연습을 할 수 있는 방법에 대해 정리합니다. 지난 회의 때 별도로 설명을 했고, 그 내용을 글로 정리한 것입니다.
해당 발표 방법을 사용하기 전에 선행되어야 하는 일이 있습니다.

- 이슈를 소단위 챕터별로 생성하기 

## 발표 방법
1. 발표자 정하기
2. 해당 발표 내용과 맞는 이슈에 5분간 `어떤 내용을 설명할지` planning
3. 20분간 발표를 진행
4. 5분간 다른 참여자가 발표에 대한 피드백을 이슈에 코멘트
5. 해당 이슈 닫기

위와 같이 진행 시 
- 발표 내용에 대해 기록할 수 있으며, 
- 해당 이슈의 링크를 챕터별 리드미에 걸어 보여줄 수 있습니다.
- 또한, 5분, 20분, 5분의 30분 단위로 루틴을 만들어 시간을 효율적으로 쓸 수 있습니다.

**Comment by Sleepingoff** [comment]
다만, 주의할 점은
- 발표와 퀴즈를 모두 20분 내로 끝내어 발표 내용이 부실할 수 있습니다.

위의 문제에도 이 방법을 추천하는 이유는
- 책에 대한 1회독을 끝내는 것이 목표이기 때문이며,
- 시간을 최대한 효율적으로 쓸 수 있기 때문입니다.

**Reply by atoonatoo** [reply]
좋은 정리입니다 ! 고생하셨어요
